lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -58,23 +57,16 @@ $(document).on('click', '.js-kudos-button', function(event) {
   const dataKid = $el.attr('data-kid');
   const method = dataKid ? 'DELETE' : 'POST';
 
-  // if (setting('dsKudosReactions.sendToRogue')) {
-  //   makeKudosRequestToRogue(setting('dsKudosReactions.northstarId'), setting('dsKudosReactions.postId'))
-  // } else {
   makeKudosRequest($el, dataKid)
     .done((response) => {
       const kid = get(response, '0.kid', '');
-
-      if (method === 'POST' && !kid) {
-        return;
-      }
-
+      // if (method === 'POST' && !kid) {
+      //   return;
+      // }
```

Comment:
Can you try logging out what the `kid` variable is when this happens? I think this should work without changes on the front end, unless perhaps we're not providing that response the way it's expected.